enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/mtls/MtlsUtils.java

```diff
@@ -0,0 +1,130 @@
+                throw new SecurityException(e);
+            }
+        }
+        if (isPublicCertificate(cert)) {
+            String content = cert.replaceAll("-----BEGIN CERTIFICATE-----", "")
+                    .replaceAll("-----END CERTIFICATE-----", "");
+            return content.trim();
+        }
+        log.debug("Provided client certificate is not a public certificate.");
+        return "";
+    }
+
+    public static KeyStore createTrustStore(List<Certificate> clientCertificates) throws KeyStoreException {
+        KeyStore trustStore = KeyStore.getInstance("JKS");
+        try {
+            trustStore.load(null, null);
+        } catch (IOException | NoSuchAlgorithmException | CertificateException e) {
+            log.debug("Creating the client certificate truststore was unsuccessful.");
+            throw new SecurityException(e);
+        }
+        X509Certificate x509cert;
+        for (Certificate certificate : clientCertificates) {
+            String alias = certificate.getAlias();
+            String cert = certificate.getContent().toStringUtf8();
+            String certContent = getCertContent(cert);
+            try {
+                x509cert = getX509Cert(certContent);
+            } catch (CertificateException e) {
+                throw new SecurityException(e);
+            }
+            trustStore.setCertificateEntry(alias, x509cert);
+        }
+        return trustStore;
+    }
+
+    public static X509Certificate getX509Cert(String certContent)
+            throws CertificateException {
+        CertificateFactory cf = CertificateFactory.getInstance("X.509");
+        byte[] bytes = Base64.decodeBase64(certContent);
+        X509Certificate x509Certificate;
+        try (ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(bytes)) {
+            x509Certificate =
+                    (X509Certificate) cf.generateCertificate(byteArrayInputStream);
+        } catch (IOException e) {
+            log.debug("Unable to generate x509 certificate format.");
+            throw new CertificateException(e);
+        }
+        return x509Certificate;
+    }
+
+    public static String getMatchedCertificateAliasFromTrustStore(X509Certificate certificate, KeyStore trustStore)
+            throws CertificateException {
+        String alias = null;
+        try {
+            if (trustStore != null) {
+                alias = trustStore.getCertificateAlias(certificate);
+            } else {
+                log.debug("The API truststore has not been initialized");
+            }
+        } catch (KeyStoreException e) {
+            log.debug("Error occurred while checking the API truststore.");
+            throw new CertificateException(e);
+        }
+        return (alias);
+    }
+
+    public static String getTier(String alias, Map<String, String> tiersMap) {
+        String tier = "";
+        tier = tiersMap.get(alias);
```

Comment:
please check other locations where there's unnecessary separation between var init and value assigning 
```suggestion
        String tier = tiersMap.get(alias);
```

Reply:
Fixed